stable/kubernetes-dashboard/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kubernetes-dashboard
-version: 1.2.2
+version: 1.3.0
```

Comment:
I don't know why CI says that the version is not bumped. Maybe I need to move this to the last commit in the PR?

Reply:
Extracted to a separate commit but it didn't help. Don't know what to do.

Reply:
another patch merged that was 1.3.0. should be 1.4.0 now. Sorry.

Reply:
Oh, I forgot to add the origin remote to my repo. Solved.